aspnetcore/blazor/state-management/persistent-component-state.md

```diff
@@ -0,0 +1,355 @@
+---
+title: ASP.NET Core Blazor persistent component state
+author: guardrex
+description: Learn how to persist user data (state) in Blazor apps using Blazor's Persistent Component State service.
+monikerRange: '>= aspnetcore-8.0'
+ms.author: wpickett
+ms.custom: mvc
+ms.date: 08/04/2025
+uid: blazor/state-management/persistent-component-state
+---
+# ASP.NET Core Blazor persistent component state
+
+Without persisting component state, state used during prerendering is lost and must be recreated when the app is fully loaded. If any state is created asynchronously, the UI may flicker as the prerendered UI is replaced when the component is rerendered.
```

Comment:
Persisting component state isn't just about prerendering. Can we lead off with a more general introduction into why persisting component state is important? It's useful for reusing the state from prerendering, pausing & resuming circuits, and maintaining state across browser refreshes.